assets/src/components/detours/diversionPage.tsx

```diff
@@ -433,7 +442,11 @@ export const DiversionPage = ({
               : "text-bg-light",
           ])}
         >
-          {detourPanel()}
+          {isMobile(displayType) ? (
+            <DrawerWithState startOpen>{detourPanel()}</DrawerWithState>
```

Comment:
https://github.com/mbta/skate/pull/2843/files#r1795948724
```suggestion
            <Drawer.WithState startOpen>{detourPanel()}</Drawer.WithState>
```

Reply:
Ok this is something I haven't learned yet and didn't know how to frame the question! But... why this?

Reply:
Ah ok, so it's a linguistic preference instead of a technical need. I'm ok with that